spec/library.rb

```diff
@@ -0,0 +1,61 @@
+require './write_file'
+require './read_file'
+require './data'
+require './book'
+require './person'
+
+class Library
+  attr_accessor :files, :data
+
+  def initialize
+    @files = @writer_file = WriteFile.new
+    @data = Data.new
```

Comment:
- [ ] I got the attached error when I tried running your tests. Kindly fix this error so that we can check if your tests pass.
<img width="385" alt="chrome_Ns3rJgws7Z" src="https://user-images.githubusercontent.com/81468675/176793330-be4c8900-b691-416c-b9c5-5ee2a090ad00.png">

Reply:
Hii @julie-ify, Thank you for your suggested test. I rerun the same test cases. It is running properly. There is no error. I will implement your suggested test. PFA of re-run test cases.
![Screenshot from 2022-07-01 16-32-13](https://user-images.githubusercontent.com/78268827/176882673-9eec3c1e-89b4-4f66-8054-25fd2d665b52.png)
![Screenshot from 2022-07-01 16-31-55](https://user-images.githubusercontent.com/78268827/176882685-9941bc86-1385-456c-8093-788c8d2552f7.png)